addons/api/tests/unit/serializers/storage-bucket-test.js

```diff
@@ -359,4 +359,46 @@ module('Unit | Serializer | storage bucket', function (hooks) {
       },
     });
   });
+
+  test('it trims whitespace on region and role_arn fields', async function (assert) {
```

Comment:
I will suggest to not create a new test case for this specific. Perhaps we can embed this 2 fields are being serialized correctly in any test we already have within this test file? Preferebly, we can modify these 2 fields within all tests that are serializing to make sure we receive the expected behaviour. I.E: in one test add only 1 white space, in other test add 2 white spaces.

Reply:
Good idea